### PR TITLE
add admob adapter

### PR DESCRIPTION
**Pull request by Mintegral-Lee**
add admob adapter.

### CODE REVIEW

File: adapters/Mintegral/MintegralAdapter/GADMediationAdapterMintegralConstants.h [hunk 1]
```diff
@@ -15,7 +15,7 @@
 #import <Foundation/Foundation.h>
 
 /// Mintegral mediation adapter version.
-static NSString *const GADMAdapterMintegralVersion = @"7.4.2.0";
+static NSString *const GADMAdapterMintegralVersion = @"7.4.3.0";
```
Comment (joshh-devrel) [inline adapters/Mintegral/MintegralAdapter/GADMediationAdapterMintegralConstants.h:18]: @Mintegral-Lee We already updated the version, can you rebase this pull request?

File: adapters/Mintegral/MintegralAdapter/Waterfall/GADMAdapterMintegralBannerAdLoader.m [hunk 2]
```diff
@@ -79,6 +79,7 @@ - (void)loadBannerAdForAdConfiguration:(nonnull GADMediationBannerAdConfiguratio
                                                    rootViewController:rootViewController];
   _bannerAdView.delegate = self;
   _bannerAdView.autoRefreshTime = 0;
+  [_bannerAdView setExtraInfo:adConfiguration.watermark forKey:@"admob_watermark"];
```
Comment (joshh-devrel) [inline adapters/Mintegral/MintegralAdapter/Waterfall/GADMAdapterMintegralBannerAdLoader.m:82]: Remove, watermark is for bidding ads only.

The same for all the other waterfall ad formats.

**Comment by joshh-devrel** [comment]
@Mintegral-Lee Can you add the below in the CHANGELOG?
```
#### Next Version
- Added watermark support for bidding ads.
- ```

File: adapters/Mintegral/MintegralAdapter/Waterfall/GADMAdapterMintegralBannerAdLoader.m (hunk 2 above)
Comment (Mintegral-Lee) [reply adapters/Mintegral/MintegralAdapter/Waterfall/GADMAdapterMintegralBannerAdLoader.m]: I've fixed it, thank you.

**Comment by Mintegral-Lee** [comment]
> @Mintegral-Lee Can you add the below in the CHANGELOG?
> 
> ```
> #### Next Version
> - Added watermark support for bidding ads.
> ```

@joshh-devrel I have added.

File: adapters/Mintegral/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 #### [Version 7.4.3.0](https://dl.google.com/googleadmobadssdk/mediation/ios/mintegral/MintegralAdapter-7.4.3.0.zip)
 - Verified compatibility with Mintegral SDK version 7.4.3.
+- Added watermark support for bidding ads.
```
Comment (joshh-devrel) [inline adapters/Mintegral/CHANGELOG.md:5]: Remove this line and add this above:
```
#### Next version
- Added watermark support for bidding ads.
```

Comment (Mintegral-Lee) [reply adapters/Mintegral/CHANGELOG.md:5]: @joshh-devrel 
I don’t understand what you mean. It is true that Mintegral SDK version 7.4.3 already supports admob watermark.

Comment (joshh-devrel) [reply adapters/Mintegral/CHANGELOG.md:5]: @Mintegral-Lee 
To clarify, Mintegral iOS adapter 7.4.3.0 is already live so this code change can't be included in it, but the next version.

**Pull request merged**